### ISSUE TITLE
Add hover colors to @carbon/colors

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#2877]
We are frequently using our hover color palette inside of our components, as a result we should add these colors to the `@carbon/colors` package here: https://github.com/carbon-design-system/carbon/blob/master/packages/colors/src/colors.js

Initial idea would be to do the following:

```js
export const red10 = '#fff0f1';
export const red20 = '#fcd0d3';
export const red30 = '#ffa4a9';
export const red40 = '#ff767c';
export const red50 = '#fb4b53';
export const red60 = '#da1e28';
export const red70 = '#a51920';
export const red80 = '#750e13';
export const red90 = '#570408';
export const red100 = '#2c080a';
export const red = {
  10: red10,
  20: red20,
  30: red30,
  40: red40,
  50: red50,
  60: red60,
  70: red70,
  80: red80,
  90: red90,
  100: red100,
};

export const red10Hover= '#fff0f1';
export const red20Hover= '#fcd0d3';
export const red30Hover= '#ffa4a9';
export const red40Hover= '#ff767c';
export const red50Hover= '#fb4b53';
export const red60Hover= '#da1e28';
export const red70Hover= '#a51920';
export const red80Hover= '#750e13';
export const red90Hover= '#570408';
export const red100Hover = '#2c080a';
export const redHover = {
  10: red10Hover,
  20: red20Hover,
  30: red30Hover,
  40: red40Hover,
  50: red50Hover,
  60: red60Hover,
  70: red70Hover,
  80: red80Hover,
  90: red90Hover,
  100: red100Hover,
};

export const colors = {
  red,
  redHover,
};
```

**Comment by joshblack** [comment]
cc @carbon-design-system/design should we look into adding the hover colors? Or is this no longer relevant?

**Comment by shixiedesign** [comment]
We are looking into dropping the hover color palette. Currently waiting on finalizing the V2 of core palette. Instead of using a whole new palette for hover colors, we'd like to use the simpler method like `lighten($blue60, 16%);`

**Comment by joshblack** [comment]
Awesome, thanks @shixiedesign!
